frontend/packages/kubevirt-plugin/src/components/modals/boot-order-modal/boot-order-modal.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Modal, Button } from '@patternfly/react-core';
+import { HandlePromiseProps, withHandlePromise } from '@console/internal/components/utils';
+import { ModalComponentProps } from '@console/internal/components/factory';
+import { k8sPatch } from '@console/internal/module/k8s';
+import { VMLikeEntityKind } from '../../../types';
+import { getDevices } from '../../../selectors/vm/combined';
+import { getVMLikeModel } from '../../../selectors/vm';
+import { PatchBuilder, PatchOperation } from '../../../k8s/utils/patch';
+import { getVMLikePatches } from '../../../k8s/patches/vm-template';
+import { BootOrder } from '../../boot-order';
+
+const modalTitle = 'Virtual machine boot order';
+
+export const BootOrderModal = withHandlePromise(
+  ({ vmLikeEntity, handlePromise, onClose, title = modalTitle }: BootOrderModalProps) => {
+    const [devices, setDevices] = React.useState<any[]>(getDevices(vmLikeEntity));
+
+    const submit = async (event) => {
+      event.preventDefault();
+
```

Comment:
maybe we should "relax" the bootorder before the request? It could be higher depending on remove/add of devices

Reply:
I added a relax code to `delete`, `add` and `move` already leave the device boot order relaxed.
so now all operations leave the list relaxed.